### PR TITLE
fix/Switch-component-to-Select

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
#### Summary
`Switch` component was changed to the `Selector` component from the `santiment/app`.
* `Selector` component API was not changed;
* Styles were refactored;
* `Selector` now has `border` variant.

#### Screenshots
image

### CODE REVIEW

**Comment by DmitriVanGuard** [comment]
@Partysun We have multiple occurrences of the component like this in our design system and there definitely should be the place for the component like this. If another product of ours will have a need to use a variation of our UI `Select` component (like `Horizontal` or `DropdownSelect`) and there will be no such - it would break the unified flow of our UI.

File: src/Selector/Selector.module.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            border: 1px solid $green;
+        }
+    }
+}
+
+:global(.night-mode) {
+    .wrapper,
+    .selected {
+        color: #a0a0a0;
+        background-color: #252932;
```
Comment (Partysun) [inline src/Selector/Selector.module.scss]: @DmitriVanGuard In design system we got dark mode colors.

File: src/Selector/Selector.module.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    &::before {
+        display: inline-block;
+        content: '';
+        width: 100%;
+        position: absolute;
+        top: -1px;
+        left: -1px;
+        bottom: -1px;
+        background: $btn-background-light-green;
+        // box-sizing: border-box;
```
Comment (Partysun) [inline src/Selector/Selector.module.scss]: @DmitriVanGuard Clean up

File: src/Selector/Selector.module.scss (hunk 1 above)
Comment (DmitriVanGuard) [reply src/Selector/Selector.module.scss]: @Partysun They are the same as the `day mode`

**Pull request merged**